### DISCUSSION TITLE
Migration Resolve completes, but column doesn't exist in database

### DISCUSSION THREAD

**Discussion by leo-petrucci** [prisma/prisma#4937] [Q&A] [answered]
I'm having some trouble with `migrate`, but I'm not sure if I'm using it correctly.

Here's my steps:

1. In development I've added a column to my schema and ran `prisma migrate dev --preview-feature`. This successfully added the column to my table
2. I then copied the name of the migration `20210110223424_added_last_updated_column` and ran `prisma migrate resolve --applied "20210110223424_added_last_updated_column" --preview-feature` on my production database. That completed successfully.
3. For good measure I ran `prisma migrate status --preview-feature` which tells me `Database schema is up to date!`
4. I then try to run `prisma studio`, but get an error telling me:

```
Message: Error in Prisma Client request: 

Error: The column `Server.lastUpdated` does not exist in the current database.
```

So obviously either the migration didn't work or I'm missing something.

Can anyone help me?

**Comment by leo-petrucci** [comment, marked as answer]
[I've currently fixed this with `db push` but I don't think that's something I should be doing in production]

**Reply by ryands17** [reply]
Hey @creativiii 👋 
You're using the incorrect command here in step 2.

The command you need to use is the following to apply all the changes you have made to the db:

```
prisma migrate deploy --preview-feature
```

You can read more about the command [here](https://www.prisma.io/docs/reference/api-reference/command-reference/#migrate-deploy).

**Reply by leo-petrucci** [reply]
Ah that makes sense. I was following [this guide](https://www.prisma.io/docs/guides/prisma-guides/prisma-migrate-guides/add-prisma-migrate-to-a-project#baseline-your-production-environment) which had no mention of using `deploy` so I assumed it was used for something else entirely.

Thank you!

**Reply by ryands17** [reply]
Ohh, that is only if you already have an existing project and want to shift to Migrate :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
